_posts/2023-02-01-opentelemetry.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: "Apache Kafka tracing with Strimzi: from OpenTracing to OpenTelemetry"
+date: 2023-02-01
+author: paolo_patierno
+---
+
+Apache Kafka is one of the most used platforms for events and data streaming to build a distributed system, by providing the backbone for microservices communication.
+One of the most complex tasks to deal with such a distributed environment, is to get insights about the flow of the data and then evaluating performance.
+This is where distributed tracing comes into the picture, by addressing different issues:
+
+* monitor the flow of events or remote procedure calls between applications.
+* profile messages latency in order to help optimizing performance.
+* provide an overall picture about how microservices communicate across boundaries.
+
+Strimzi have been provided tracing support, for Kafka Connect, Kafka Mirror Maker(s) and the HTTP bridge, since the 0.14.0 release leveraging the [OpenTracing](https://opentracing.io/) project. Tracing is anyway not supported for Kafka brokers.
```

Comment:
Not really sure about how this sounds: `Tracing is anyway not supported for Kafka brokers.` ... the `anyway` seems weird there.

Reply:
Is `OpenTracing` a proper word instead of Tracing?

Reply:
> Not really sure about how this sounds: Tracing is anyway not supported for Kafka brokers. ... the anyway seems weird there.

I removed `anyway`.

> Is OpenTracing a proper word instead of Tracing?

We are talking about tracing in general here.